### PR TITLE
added acala euphrates adapter

### PR DESCRIPTION
**Pull request by shunjizhan**
##### Name (to be shown on DefiLlama):
Acala Euphrates

##### Twitter Link:
https://twitter.com/AcalaNetwork

##### List of audit links if any:
https://github.com/AcalaNetwork/Euphrates/blob/master/audit/PeckShield-Audit-Report-Acala-Euphrates-v1.0.pdf

##### Website Link:
https://farm.acala.network/

##### Logo (High resolution, will be shown with rounded borders):
Euphrates Logo black

##### Current TVL:
$10.85 M

##### Chain:
Acala

##### Short Description (to be shown on DefiLlama):
Deposit DOT liquidity and earn boosted rewards

##### Category (full list at https://defillama.com/categories) *Please choose only one:
Yield

##### methodology (what is being counted as tvl, how is tvl being calculated):
total dot/lcdot staked - total dot/lcdot unstaked

##### Github org/user (Optional, if your code is open source, we can track activity):
https://github.com/AcalaNetwork/Euphrates/tree/master

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/acala-euphrates exports TVL: 
        
 
 
    acala                     10.80 M
    
    total                    10.80 M

File: projects/acala-euphrates/index.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            smallestDiff = diff;
+            targetDotTVL = entry.dot_tvl;
+        }
+    });
+
+    return targetDotTVL;
+}
+
+async function tvl(timestamp, _1, _2, { api }) {
+    const response = await axios.get('https://api.dune.com/api/v1/query/3132389/results?api_key=ItzgVHuqp1idtYFmDAzo15kEDILs1qXb');
```
Comment (shunjizhan) [inline projects/acala-euphrates/index.js:33]: although this is a free api key from dune, is there a better way to handle this? Like putting this in `env` maybe?

**Comment by llamatester** [comment]
The adapter at projects/acala-euphrates exports TVL: 
        
 
 
    acala                     10.82 M
    
    total                    10.82 M

File: projects/acala-euphrates/index.js (hunk 1 above)
Comment (g1nt0ki) [reply projects/acala-euphrates/index.js:33]: can you update adapter to pull the info on chain? dont like the idea of depending on third party api

**Comment by llamatester** [comment]
The adapter at projects/acala-euphrates exports TVL: 
        
 
 
    acala                     15.74 M
    
    total                    15.74 M

**Pull request merged**